test/performance/odp_sched_perf_run.sh

```diff
@@ -35,6 +38,18 @@ run()
 			echo odp_sched_perf FAILED
 			exit $RET_VAL
 		fi
+
+		echo odp_sched_perf -p 2 -c $1
+		echo ===============================================
+		$TEST_DIR/odp_sched_perf${EXEEXT} -p 2 -c $1
+		RET_VAL=$?
+		if [ $RET_VAL -eq 77]; then
+			echo "odp_sched_perf -p 2 -c $1 skipped (event vectors not supported)"
+			exit $TEST_SKIPPED
```

Comment:
Since we want to still continue running the tests for other worker counts with packets and buffers, the script should not exit here. Printing the message should be enough.